EnglishFilter/sections/specific.txt

```diff
@@ -2368,7 +2368,11 @@ heroero.com##.block-video > .table > .opt
 ||heroero.com/temp/2ava.jpg
 efukt.com##.efukt-widget-slider-nice-try-adblockers
 sportnews.to##span.mol-ads-label-container
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com,important
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com|onlinemschool.com,important
```

Comment:
According to whitelist comment blocking that element causes infinite loading. If no infinite loading happens with that rule, please remove whitelist rule.

Left-over for desktop:
#oms_res_block  > div[id^="oms_oag"][style="text-align:center;margin:1em 0;"]